tests/main/security-device-cgroups-devmode/task.yaml

```diff
@@ -31,7 +40,7 @@ execute: |
     test-devmode-cgroup.read-dev fb0 2>&1 | MATCH -v '(Permission denied|Operation not permitted)'
 
     echo "the devmode snap can access other devices"
-    test-devmode-cgroup.read-dev mem
+    test-devmode-cgroup.read-dev test1 | MATCH -v '(Permission denied|Operation not permitted)'
```

Comment:
Can you check if those kernels have CONFIG_STRICT_DEVMEM? eg `zgrep /proc/config.gz`

Reply:
Yes, but for this test I think best solution is to use a fake char davice which we know in all the images/boards, if I make that check I'll end up skipping the validation on some boards. Does it make sense?

Reply:
Sorry for adding confusion. I was just curious whether `CONFIG_STRICT_DEVMEM` is enabled in those kernels and I suspected it could be causing the issues with `/dev/mem`.

Reply:
If I remember correctly CONFIG_STRICT_DEVMEM is enable on rpi kernel. I'll manually check it today.